### PR TITLE
test updates

### PR DESCRIPTION
**Pull request by ginberg**
- test updates 1.24.8

### CODE REVIEW

File: tests/testthat/test-ui-bar.R [hunk 1]
```diff
@@ -41,10 +43,12 @@ test_that("cXbar10", {
     check_ui_test(cXbar10())
 
     warning("Durable overlay looks different in R than on the CX website, missing the following parm: smpOverlayProperties=list('Durable' = list('type'='Boolean'))")
+    warning("When clicking on Durable with value = 0 (e.g. at S8), all bars are disabled")
```
Comment (cb4ds) [inline tests/testthat/test-ui-bar.R]: This is only a problem if you don't follow the first warning and add the missing parameter for the test - if you do this you can't actually turn "off" the boolean zero value, so this doesn't apply.  Please remove the second warning.

File: tests/testthat/test-ui-bar.R [hunk 2]
```diff
@@ -41,10 +43,12 @@ test_that("cXbar10", {
...
 })
 
 test_that("cXbar11", {
     check_ui_test(cXbar11())
 
     warning("Durable overlay looks different in R than on the CX website, missing the following parm: smpOverlayProperties=list('Durable' = list('type'='Boolean'))")
+    warning("When clicking on Durable with value = 0 (e.g. at S8), all bars are disabled")
```
Comment (cb4ds) [inline tests/testthat/test-ui-bar.R]: This is only a problem if you don't follow the first warning and add the missing parameter for the test - if you do this you can't actually turn "off" the boolean zero value, so this doesn't apply.  Please remove the second warning.

Comment (ginberg) [reply tests/testthat/test-ui-bar.R]: Ah I see, I will update it
